sycl/include/CL/sycl/platform.hpp

```diff
@@ -126,6 +126,22 @@ class __SYCL_EXPORT platform {
   /// given feature.
   bool has(aspect Aspect) const;
 
+  /// Return this platform's default context
+  ///
+  /// \return the default context
+  context get_default_context() const;
+
+  /// Pushes a new default context
+  ///
+  /// \param the new default context
+  void push_default_context(context Context);
```

Comment:
```suggestion
  void push_default_context(const context &Context);
```
Can we change it to take a reference?